extensions/wikia/AdEngine/js/slot/slotTargeting.js

```diff
@@ -106,8 +107,18 @@ define('ext.wikia.adEngine.slot.slotTargeting', [
 		}
 	}
 
+	function getOutstreamData() {
+		var getVideoTargeting = prebid.get().getAdserverTargetingForAdUnitCode,
+			videoTargeting = getVideoTargeting && getVideoTargeting('INCONTENT_PLAYER');
```

Comment:
Is it ok that we limit only to this slot? Video is also displayed in TLM and MIC